packages/script/install.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e -x
+getent passwd yugabyte > /dev/null 
+
+if [ $? -eq 0 ]; then
```

Comment:
we never hit these lines because we have set -e on the script. when the getent fails, the script bails out immediately.